lib/puppet/parser/functions/pick.rb

```diff
@@ -6,9 +6,11 @@
 module Puppet::Parser::Functions
   newfunction(:pick, type: :rvalue, doc: <<-EOS
     @summary
-      This function is similar to a coalesce function in SQL in that it will return
+      This function will return
       the first value in a list of values that is not undefined or an empty string.
```

Comment:
Perhaps even shorten it to "Return the first ..."?

Reply:
I tried to use the same phrasing as used in the other functions (they start with "This function").  It is not very consistent so maybe somebody will step-in to improve phrasing and consistency. As a non-native English speaker, I do not feel confident for such a task :fearful: